packages/flutter_tools/lib/src/error_messages.dart

```diff
@@ -0,0 +1,177 @@
+import 'base/context.dart';
+import 'ios/cocoapods.dart';
+import 'vscode/vscode_validator.dart';
+
+ErrorMessages get errorMessages => context[ErrorMessages];
+
+/// Class containing the messages that can be produced by the doctor validators.
```

Comment:
I think this class can be more general than just for validators - we could grow it to use it for thrown exceptions as well.

Reply:
That seems likely. Changed the doc comment to be more general.